assignments/function-conversion.js

```diff
@@ -5,21 +5,41 @@
 // };
 // myFunction();
 
+ myFunction = ('Function was invoked!');
+    console.log(myFunction)
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
-// anotherFunction("Example");
+// anotherFunction("Example")
+anotherFunction = ("Example") 
+
```

Comment:
I am curious as to why this and the last challenge are just written as variables rather than actual function while the ones below are? 

```js
//              Assigned to      Fat Arrow indicating funtion
//                    v          v      
  let anotherFunction = (param) => { return param };
//       ^                 ^              ^
//  Variable Name      Parameters       Function body
```

There is some nuance to the syntax like:

If there is only 1 parameter the `()` are not necessary, but if there are none, or more than 1 they are.

If there is no `{}` around the function body, it will implicitly return whatever comes after the Fat Arrow and do nothing else. 

Could that be what had caused some confusion on these last 2?